test/nyc-bin.js

```diff
@@ -71,6 +71,30 @@ describe('the nyc cli', function () {
     })
   })
 
+  describe('--ignore-class-method', function () {
+    if (parseInt(process.versions.node.split('.')[0]) < 4) return
```

Comment:
I don't think this check should be needed; we dropped Node <4 support in CI a few months ago.